avocado/utils/linux_modules.py

```diff
@@ -96,7 +101,23 @@ def loaded_module_info(module_name):
     :rtype: dict
```

Comment:
`:return:` is outdated: `Dictionary of module info, name, size, submodules if present`

While in the dict:
```
>>> pprint(loaded_module_info('kvm_intel'))
{'depends': ['kvm'],
 'filename': '/lib/modules/4.8.13-300.fc25.x86_64/kernel/arch/x86/kvm/kvm-intel.ko.xz',
 'name': 'kvm_intel',
 'params': ['vpid',
            'flexpriority',
            'ept',
            'unrestricted_guest',
            'eptad',
            'emulate_invalid_guest_state',
            'vmm_exclusive',
            'fasteoi',
            'enable_apicv',
            'enable_shadow_vmcs',
            'nested',
            'pml',
            'preemption_timer',
            'ple_gap',
            'ple_window',
            'ple_window_grow',
            'ple_window_shrink',
            'ple_window_max'],
 'size': 188416,
 'submodules': [],
 'used': 0}

```